generators/entity/files.js

```diff
@@ -213,43 +213,43 @@ const angularFiles = {
             templates: [
                 {
                     file: 'entities/_entity-management.component.html', method: 'copyHtml', template: true,
-                    renameTo: generator => `entities/${generator.entityFolderName}/${generator.entityPluralFileName}.component.html`
+                    renameTo: generator => `entities/${generator.entityFolderName}/${generator.entityFileName}-management.component.html`
```

Comment:
the management word is the angularSuffix and hence this should be only `entityFileName` here. remove the management